docs/reference/outlook.md

```diff
@@ -0,0 +1,140 @@
+# Outlook Connector
+
+The [Elastic Outlook connector](../connectors/sources/outlook.py) is built with the Elastic connectors Python framework and is available as a self-managed [connector client](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+## Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+## Usage
+
+To use this connector as a **connector client**, use the **Outlook** tile from the connectors list OR **Customized connector** workflow.
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+## Connecting to Outlook
+
+### Connect to Exchange Server
+
+To order to connect to Outlook, the connector fetches Active Directory users with the help of `ldap3` python library.
+
+### Connect to Office365 Outlook
+
+To order to connect to Outlook, user needs to create an Azure application as follows:
+
+1. Log into Microsoft [Azure Portal](https://portal.azure.com) and click on App registrations and register an application.
+2. From Overview tab, keep `Client ID` and `Tenant ID` handy.
+3. Go to **Certificates & secrets** and create a new client secret.
+4. Before configuring `Client ID`, `Tenant ID` and `Client Secret` in elastic, from API permissions tab - add below permission to fetch the Outlook content,
```

Comment:
This is a little confusing, can't we simply say 'Add the following permissions in the **API permissions tab**...'?